platform-sdk/docs/proposals/TSS-Encryption-Key-Automation/TSS-Encryption-Key-Automation.md

```diff
@@ -178,17 +173,18 @@ message TssEncryptionKeyTransaction {
 
 #### TssBaseService New State
 
-The `TssBaseService` state will be extended with a new map of the public `tssEncryptionKey` for each node.
+The `TssBaseService` state will be extended with a new virtual map of the public `tssEncryptionKey` for each node.
 
-1. The keys of the map are Longs/UInt64 for the `nodeId` of the nodes
+1. The keys of the map are UInt64 for the `nodeId` of the nodes
```

Comment:
Minor: while personally I like longs/uint64 too, I've heard many people find it useful and more type-safe to actually use a "node id" abstraction for representing node ids. So if we already have a protobuf that represents a node id, we might as well use it here, too.

Reply:
Node ID is a `uint64` everywhere.

Reply:
Good to know. Thanks for confirming.